src/v2/cookbook/serverless-blog.md

```diff
@@ -0,0 +1,268 @@
+    {
+      path: '/blog/',
+      name: 'blog-home',
+      component: BlogHome
+    },
+    {
+      path: '/blog/:slug',
+      name: 'blog-post',
+      component: BlogPost
+    }
+  ]
+})
+```
+
+Then create `components/BlogHome.vue` which will be your blog homepage that lists your most recent posts.
+
+```javascript
+<script>
+  import { butter } from '@/buttercms'
+  export default {
+    name: 'blog-home',
+    data() {
+      return {
+        page_title: 'Blog',
+        posts: []
+      }
+    },
+    methods: {
+      getPosts() {
+        butter.post.list({
+          page: 1,
+          page_size: 10
+        }).then((res) => {
+          // console.log(res.data)
+          this.posts = res.data.data
+        })
+      }
+    },
+    created() {
+      this.getPosts()
+    }
+  }
+</script>
+Display the result
+
+<template>
+  <div id="blog-home">
+      <h1>{{ page_title }}</h1>
+      <!-- Create v-for and apply a key for Vue. Example is using a combination of the slug and index -->
+      <div v-for="(post,index) in posts" :key="post.slug + '_' + index">
+        <router-link :to="'/blog/' + post.slug">
+          <article class="media">
+            <figure>
+              <!-- Bind results using a ':' -->
+              <!-- Use a v-if/else if their is a featured_image -->
+              <img v-if="post.featured_image" :src="post.featured_image" alt="">
+              <img v-else src="http://via.placeholder.com/250x250" alt="">
+            </figure>
+            <h2>{{ post.title }}</h2>
+            <p>{{ post.summary }}</p>
+          </article>
+        </router-link>
+      </div>
+  </div>
+</template>
+```
+
+Now create `components/BlogPost.vue` which will be your Blog Post page to list a single post.
+
+```javascript
+<script>
+  import { butter } from '@/buttercms'
+  export default {
+    name: 'blog-post',
+    data() {
+      return {
+        post: {}
+      }
+    },
+    methods: {
+      getPost() {
+        butter.post.retrieve(this.$route.params.slug)
+          .then((res) => {
+            // console.log(res.data)
+            this.post = res.data
+          }).catch((res) => {
+            console.log(res)
+          })
+      }
+    },
+    created() {
+      this.getPost()
+    }
+  }
+</script>
+Display the results
+<template>
+  <div id="blog-post">
+    <h1>{{ post.data.title }}</h1>
+    <h4>{{ post.data.author.first_name }} {{ post.data.author.last_name }}</h4>
+    <div v-html="post.data.body"></div>
+
+    <router-link v-if="post.meta.previous_post" :to="/blog/ + post.meta.previous_post.slug" class="button">
+      {{ post.meta.previous_post.title }}
+    </router-link>
+    <router-link v-if="post.meta.next_post" :to="/blog/ + post.meta.next_post.slug" class="button">
+      {{ post.meta.next_post.title }}
+    </router-link>
+  </div>
+</template>
+```
+
+Now our app is pulling all blog posts and we can navigate to individual posts. However, our next/previous post buttons are not working.
```

Comment:
It might be nice to add a screenshot here so that people can visualize what you're referring to